adonis-typings/mail.ts

```diff
@@ -24,45 +24,57 @@ declare module '@ioc:Adonis/Addons/Mail' {
    * Attachment options
    */
   export type AttachmentOptionsNode = {
-    filename?: string,
-    href?: string,
-    httpHeaders?: { [key: string]: any },
-    contentType?: string,
-    contentDisposition?: string,
-    encoding?: string,
-    headers?: { [key: string]: any },
+    filename?: string;
```

Comment:
Any reason for adding semi-colon here?

Reply:
Actually, it's ts-lint that added the semi-colons

Reply:
Can you share the command you ran to format the code using `tslint`?

Reply:
At first, I configure the `tslint` extension to format on save. Running `npm run lint` also resulted in the same formatting.

Reply:
Can you share the setting for same? Trying to reproduce it on my end and prevent this from happening

Reply:
```
"editor.codeActionsOnSave": {
    "source.fixAll.tslint": true
  },
```

By default, I'm guessing `tslint` will default to the project's `tslint.json`

Reply:
Do you aslo have prettier running @ammezie?

Reply:
Yes, I think

Reply:
@RomainLanz are you saying I should disable prettier?

Reply:
@thetutlage I have removed the semi-colons.  I believe I still need to tweak vscode to prevent this going forward.

Reply:
Yes, Prettier is taking over any configuration we have, and we cannot do much about it. You need to configure Prettier on your side to not be turned on if it didn't find a config inside the workspace.

Reply:
Okay.